x-pack/plugins/security/server/routes/views/logout.js

```diff
@@ -10,7 +10,7 @@ export function initLogoutView(server) {
   server.route({
     method: 'GET',
     path: '/logout',
-    handler(request, h) {
+    handler(request, h) { // eslint-disable-line no-unused-vars
```

Comment:
`request` is unused, but I'm not sure how else we'd deal with this. also, linting passes without the ignore, why are we adding it?

Reply:
It's a good question. Normally I wouldn't fuss over such things.

I'm attempting to smooth the migration to the new platform and managing dependencies is central to that effort. Angular provides dependency injection and we have a fair amount of code where injected dependencies are unused. This adds a lot of noise when attempting to reason about injected dependencies. A cleanup pass is required. 

With the correct settings ESLint very helpfully highlights unused function params but its not necessarily clear whether they're angular injected dependencies or not. I have a list of about 800 instances of unused params and quite a few are angular dependencies. I can't spend much time on any given bit of code if I'm going to make progress.

Generally, I think people are happy to see the cleanup effort. Unless there are objections, I'll go ahead an implement stricter eslint settings once all errors are addressed. Even if someone objects I'm able to track my progress while running the stricter settings locally.  Hopefully I'm not causing trouble making these changes.

I should probably put this into an issue and reference this in PRs. I'm happy to have further discussions if you'd like.

Reply:
Gotcha, thanks for the context. Is it possible to only scope this additional linting only to AngularJS code, or at least only to public code?

Reply:
I'm not sure how to scope for angular code. Obviously I could ignore the `x-pack` dir.